src/Svelte/assets/test/render_controller_without_props.test.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import { getByTestId, waitFor } from '@testing-library/dom';
+import { clearDOM, mountDOM } from '@symfony/stimulus-testing';
+import { startStimulus } from './util/render_controller';
+
+describe('SvelteController without props', () => {
+
+    it('connect without props', async () => {
+
+        const container = mountDOM(`
+          <div data-testid="component" id="container"
+              data-controller="check svelte"
+              data-svelte-component-value="SvelteComponent" />
+        `);
+
+        const component = getByTestId(container, 'component');
+        expect(component).not.toHaveClass('connected');
+        expect(component).not.toHaveClass('mounted');
+
+        await new Promise(resolve => setTimeout(resolve, 1000));
+        startStimulus();
+        await waitFor(() => expect(component).toHaveClass('connected'));
+        await waitFor(() => expect(component).toHaveClass('mounted'));
+        await waitFor(() => expect(component.innerHTML).toEqual('<div><div>Hello without props</div></div>'));
+
+        clearDOM();
+    });
+});
```

Comment:
Let's combine these 2 test files into one - probably we just need one `describe()` and then 2 `it()`. Then we can remove the `test/util/render_controller.ts` file and embed that logic into the one test file.

Reply:
I merged them ! Initially I had a problem because I was not stopping Stimulus after each test, so I separated them to get around the problem 😇